apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -185,7 +185,13 @@
     }
 %>
 
-<% if (Boolean.parseBoolean(loginFailed) && !errorCode.equals(IdentityCoreConstants.USER_ACCOUNT_NOT_CONFIRMED_ERROR_CODE)) { %>
+<% if (StringUtils.equals(request.getParameter("errorCode"), IdentityCoreConstants.USER_ACCOUNT_LOCKED_ERROR_CODE) &&
+        StringUtils.equals(request.getParameter("remainingAttempts"), "0") ) { %>
+<div class="ui visible negative message" id="error-msg" data-testid="login-page-error-message">
+    <%=AuthenticationEndpointUtil.i18n(resourceBundle, "error.user.account.locked.incorrect.login.attempts")%>
+</div>
+<% } else if (Boolean.parseBoolean(loginFailed) &&
+        !errorCode.equals(IdentityCoreConstants.USER_ACCOUNT_NOT_CONFIRMED_ERROR_CODE)) { %>
```

Comment:
Is this null safe?

Reply:
Yes, Initially we set the `logineFailed` value to `false` https://github.com/wso2/identity-apps/blob/705da3688f39b8b0d1b304fc18cd6f6fa63127c4/apps/authentication-portal/src/main/webapp/login.jsp#L64